src/model-helpers/DocumentList.js

```diff
@@ -0,0 +1,97 @@
+import {
+  getSolidDataset,
+  setThing,
+  getThingAll,
+  saveSolidDatasetAt,
+  createSolidDataset,
+  saveFileInContainer,
+  getThing,
+  removeThing,
+  deleteFile
+} from '@inrupt/solid-client';
+
+import { makeDocIntoThing, parseDocFromThing } from './Document';
+
+const parseFromDataset = async (docsDataset, session) => {
```

Comment:
Think we can get away with dropping the async/await completely in this function. None of the get functions require using session, which does the fetch via session.fetch so we don't have to deal with any promises.

Reply:
fixed